lib/m2ts/caption-stream.js

```diff
@@ -99,65 +99,57 @@ CaptionStream.prototype.push = function(event) {
   this.latestDts_ = event.dts;
 };
 
-CaptionStream.prototype.sortAndDispatchPackets = function(packets) {
+CaptionStream.prototype.flushStream = function(flushType) {
+  if (flushType !== 'flush' && flushType !== 'partialFlush') {
```

Comment:
I don't know if we need this guard here, since we shouldn't encounter any other types (and tests should verify that we are making the call correctly in either case).